content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,271 @@
+   of the :guilabel:`Booths` page.
+
+The booths present in the :guilabel:`Available` stage are still available for people to purchase for
+the event. The booths present in the :guilabel:`Unavailable` stage have already been purchased, and
+are no longer available.
+
+To modify any existing booth, simply click the desired booth from the :guilabel:`Booths` page, and
+proceed to make any necessary changes from the booth form. Or, create a new one, by clicking the
+:guilabel:`New` button in the upper-left corner to reveal a blank booth form.
+
+Booth form
+----------
+
+The booth form in Odoo *Events* lets users customize and configure event booths in a number of
+different ways.
+
+.. image:: event_booths/booth-form.png
+   :align: center
+   :alt: Typical booth form in the Odoo Events application.
+
+Start by typing in a :guilabel:`Name` for the booth. This is a **required** field.
+
+Then, apply a :guilabel:`Booth Category` to the booth. This is a **required** field.
+
+.. tip::
+   A new :guilabel:`Booth Category` can be created from this field, by typing in the name of the
+   new category, and clicking :guilabel:`Create and edit...` from the resulting drop-down menu.
+   Doing so reveals a :guilabel:`Create Booth Category` pop-up window, with all the standard fields
+   found on a common booth category form.
+
+   Simply clicking :guilabel:`Create` from the resulting drop-down menu creates the category, but
+   does not reveal the :guilabel:`Create Booth Category` pop-up window. The category would have to
+   be modified later, via the *Booth Categories* page (:menuselection:`Events app --> Configuration
+   --> Booth Categories`).
+
+Upon selecting a pre-existing :guilabel:`Booth Category`, two additional, non-modifiable fields
+appear: :guilabel:`Product` and :guilabel:`Price`. Both fields represent their respective selections
+for that specific booth category.
+
+When a person purchases a booth rental through the event website, the subsequent renter-related
+fields on the form auto-populate, based on the information provided by the purchaser during the
+online transaction. The booth also automatically changes its status from *Available* to
+*Unavailable*.
+
+However, if the rental of a booth is conducted in any other way (e.g., in person, via sales order,
+etc.), the :guilabel:`Renter`, :guilabel:`Renter Name`, :guilabel:`Renter Email`, and
+:guilabel:`Renter Phone` fields can be entered in manually.
+
+The status of the booth (:guilabel:`Available` or :guilabel:`Unavailable`) can also be changed
+manually, either by clicking the appropriate status from the status bar present on the booth form,
+or by dragging-and-dropping the desired booth into the appropriate stage, via the *Booths* page
+Kanban view.
+
+Sell event booths
+=================
+
+With event booths configured in the back-end, via the event-specific *Booths* pages, Odoo presents
+them on the front-end of the event website, via the *Get A Booth* event subheader link.
```

Comment:
I'm going to nitpick for a bit:

While it's FANTASTIC that you've been deepening your knowledge in developer concepts, terms like "front-end" and "backend" are typically used in programming to refer to server-side code (backend) and webpage-side code (front-end).
![image](https://github.com/odoo/documentation/assets/23611711/e718494c-cf69-4479-b76b-184c8f41e292)
[For more info, here's an article](https://aws.amazon.com/compare/the-difference-between-frontend-and-backend/)

Odoo simplifies website development by allowing users to modify webpages through the Odoo app, removing the need for code. Because of this abstraction, it's no longer suitable to refer to what we do in Odoo as front and back-end.

For our non-developer audience, it might be more technically correct to use:
backend --> event form
front-end --> event website preview

If you agree with my feedback, I'd use CTRL+F to find all instances of "backend" and "frontend" and make the adjustments :blush: